src/status_im/ios/core.cljs

```diff
@@ -23,34 +23,32 @@
   (dispatch [:shake-event]))
 
 (defn app-root [props]
-  (let [keyboard-height (subscribe [:keyboard-height])]
-    (reagent/create-class
-     {:component-did-mount
-      (fn [this]
-        (.addListener react/keyboard
-                      "keyboardWillShow"
-                      (fn [e]
-                        (let [h (.. e -endCoordinates -height)]
-                          (when-not (= h @keyboard-height)
-                            (dispatch [:set :keyboard-height h])
-                            (dispatch [:set :keyboard-max-height h])))))
-        (.addListener react/keyboard
-                      "keyboardWillHide"
-                      #(when-not (= 0 @keyboard-height)
-                         (dispatch [:set :keyboard-height 0])))
-        (.hide react/splash-screen)
-        (.addEventListener react/app-state "change" app-state-change-handler)
-        (.addEventListener rn-dependencies/react-native-languages "change" on-languages-change)
-        (.addEventListener rn-dependencies/react-native-shake
-                           "ShakeEvent"
-                           on-shake)
-        (dispatch [:set-initial-props (reagent/props this)]))
-      :component-will-unmount
-      (fn []
-        (.removeEventListener react/app-state "change" app-state-change-handler)
-        (.removeEventListener rn-dependencies/react-native-languages "change" on-languages-change))
-      :display-name "root"
-      :reagent-render views/main})))
+  ;;TODO fix that for all platforms, because this sub is always running
```

Comment:
Can we have one app-root and init for all platforms and call platform-specific under `when platform/*`? Seems like mostly it is the same code with minor differences.